app/uk/gov/hmrc/agentclientmandate/config/ScheduledJobStarter.scala

```diff
@@ -24,27 +24,29 @@ import play.api.inject.ApplicationLifecycle
 import uk.gov.hmrc.agentclientmandate.services.MandateUpdateService
 import uk.gov.hmrc.agentclientmandate.utils.LoggerUtil.{logError, logInfo, logWarn}
 import uk.gov.hmrc.play.scheduling.{ExclusiveScheduledJob, ScheduledJob}
-
-import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration.{FiniteDuration, _}
 import scala.concurrent.{Await, ExecutionContext, Future}
 import scala.language.postfixOps
 import scala.util.{Failure, Success}
 
 class DefaultScheduledJobStarter @Inject()(val app: Application,
                                            val applicationLifecycle: ApplicationLifecycle,
-                                           val mandateUpdateService: MandateUpdateService) extends ScheduledJobStarter {
+                                           val mandateUpdateService: MandateUpdateService,
+                                           implicit val ec : ExecutionContext) extends ScheduledJobStarter {
   override val scheduledJobs: Seq[ScheduledJob] = Seq(new ExclusiveScheduledJob {
     override def name: String = "ExpirationService"
+
     override def executeInMutex(implicit ec: ExecutionContext): Future[Result] = {
       mandateUpdateService.checkStaleDocuments()
       Future.successful(Result("[executeInMutex] Checking expiry of stale documents"))
     }
+
     override def interval: FiniteDuration = 1 day
+
     override def initialDelay: FiniteDuration = 0 seconds
   })
 
-  class RunnableTask(job: ScheduledJob) extends Runnable {
+  class RunnableTask(job: ScheduledJob)(implicit val ec: ExecutionContext) extends Runnable {
```

Comment:
```suggestion
  class RunnableTask(job: ScheduledJob) extends Runnable {
```
The code in the `Runnable` will get the same execution context from the implicit field on `DefaultScheduledJobStarter` anyway.